runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunnerRegistrar.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.beam.runners.flink;
```

Comment:
@ravwojdyla Perhaps you could explicitly state here that you're putting the work under the Apache license. Just to make sure we're doing everything correct here.